tests/specs/playwright/.auth/credentials.json

```diff
@@ -0,0 +1 @@
+{"email":"Art.Goyette77@yahoo.com","password":"whSneqH707MPwbp"}
```

Comment:
Be careful when exposing sensitive data such as username/password. The user has to be dynamic in every test.
With cypress we wrote the generated user and password dynamically in a json so that after registering the user, the other tests could use the login and re-run the test